x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/doc_count_chart/doc_count_chart.tsx

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useEffect, useState, useCallback, useRef, useMemo } from 'react';
+import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+import type { IImporter } from '@kbn/file-upload-plugin/public';
+import moment, { type Moment } from 'moment';
+import { useTimeBuckets } from '../../../common/hooks/use_time_buckets';
+import { IMPORT_STATUS, type Statuses } from '../import_progress';
+import { EventRateChart, type LineChartPoint } from './event_rate_chart';
+import { runDocCountSearch } from './doc_count_search';
+
+const BAR_TARGET = 150;
```

Comment:
Is this what we use in the index data visualizer? I wonder if we should look to use the `histogram:barTarget` setting, but that defaults to 50.

Reply:
I chose 150 as I thought it looked the best. I'd argue that we don't necessarily need to use `histogram:barTarget` here because this isn't something the user is going to be interacting with, it's just a "fancy progress bar".

**150** 
<img width="1185" alt="image" src="https://github.com/elastic/kibana/assets/22172091/70a8e5f3-654a-4015-bbc3-8fe82bec5712">

**100**
<img width="1187" alt="image" src="https://github.com/elastic/kibana/assets/22172091/35aa2aae-e3ad-4c15-88da-084d6bb6be26">


**50**
<img width="1187" alt="image" src="https://github.com/elastic/kibana/assets/22172091/2f2dee6b-6b1c-46c0-b070-0bf5e471078b">